docs/release-source/release/spies.md

```diff
@@ -32,6 +32,40 @@ handles a callback, as in the following simplified example:
 }
 ```
 
+### Using a spy to wrap all object method
+
+`sinon.spy(object)`
+
+Spies all the object's methods.
+
+Note that it's usually better practice to spy individual methods, particularly on objects that you don't understand or control all the methods for (e.g. library dependencies).
+
+spying individual methods tests intent more precisely and is less susceptible to unexpected behavior as the object's code evolves.
```

Comment:
Sorry for nitpicking, but I think the first character here needs to be in uppercase.